src/tests/functional/plugin/shared/include/multi/multi_remote_blob_multidevice_test.hpp

```diff
@@ -20,7 +20,8 @@ TEST_P(MultiDeviceMultipleGPU_Test, canCreateRemoteTensorThenInferWithAffinity)
     auto function = p.build();
     ov::CompiledModel exec_net;
     try {
-        exec_net = ie.compile_model(function, device_names, ov::hint::allow_auto_batching(false));
+        exec_net = ie.compile_model(function, device_names, {ov::hint::allow_auto_batching(false),
+            ov::hint::performance_mode(ov::hint::PerformanceMode::UNDEFINED)});
```

Comment:
revert this line, no need in current master

Reply:
Updated, revert line.